setup.sh

```diff
@@ -49,7 +49,7 @@ fi
 PY_VERSION=${PY_VERSION##* }
 PY_VERSION=${PY_VERSION%.*}
 # shellcheck disable=SC2072
-if [ "$PYTHON_VERSION" \< '3.12' ]; then
+if [[ ! "$PY_VERSION" > '3.11' ]]; then
```

Comment:
Is the < intentionally changed to > 3.11? Isn't this code intended to run only when python < 3.12?

Reply:
Thanks Samuel, makes sense. I guess we can go with this if needed, but it's a bit confusing like you mentioned. Would this work instead? It's clearer:

```
if [[ '3.12' > "$PY_VERSION" ]]; then
```